test_runner/src/main/kotlin/ftl/args/AndroidArgs.kt

```diff
@@ -58,6 +61,21 @@ data class AndroidArgs(
 ) : IArgs by commonArgs {
     companion object : AndroidArgsCompanion()
 
+    @get:JsonIgnore
+    val customSharding: Map<String, AndroidTestShards> by lazy {
+        if (shardingJson.isNullOrBlank()) emptyMap()
+        else {
+            fromJson<Map<String, AndroidTestShards>>(
+                Paths.get(shardingJson).toFile().readText()
+            ).mapValues { (_, shards) ->
+                shards.copy(
+                    app = shards.app.normalizeFilePath(),
+                    test = shards.test.normalizeFilePath()
+                )
+            }
+        }
+    }
+
```

Comment:
I don't like lazy properties and logical operations inside Args data classes. I know that IosArgs have one, but it should be refactored. 
Is there any reason why this logic is placed and lazily evaluated here instead of just evaluating it inside the createAndroidArgs function?

Reply:
No, there is none. PR is still in draft state so there might be (and most likely will) some changes. Anyway, thanks for input 👍🏻